### ISSUE TITLE
Support for 2stroke engines?

### ISSUE THREAD

**Issue by BasHeemskerk** [ange-yaghi/engine-sim#231]
Are 2 stroke engines ever planned? Since the technique is ofcourse completely different.

**Comment by Jckf** [comment]
They're not so different. In essence all that's happening is that the crank rotates once per engine cycle instead of twice. Someone else will have to comment on how easily this would be to implement in the sim though.

**Comment by BasHeemskerk** [comment]
Yes but the cylinder is filled with ports instead of valves, and different kinds of exhausts also respond differently due to other types of physics going on. It would be very interesting to see that.

**Comment by Evil-Spirit** [comment]
https://github.com/ange-yaghi/engine-sim/pull/309
